content/topics/dotenv-module/_index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Introduction to Dotenv module 
+ready: true
+
+---
+
+## What is an environment variable?
+
+An [environment variable](https://en.wikipedia.org/wiki/Environment_variable) is a variable whose value is set outside the program, typically through functionality built into the operating system or microservice. An environment variable is made up of a name/value pair, and any number may be created and available for reference at a point in time.
+
+```
+//example 
+NAME_OF_PLACE=Umuzi
+
+```
+
+## How does dotenv help you?
+
+Using environment variables is one technique to make your app easier to configure by separating infrequently changing data from your code. [Dotenv](https://www.npmjs.com/package/dotenv) module loads environment variables from a .env file that you create and adds them to the [process.env](https://nodejs.org/docs/latest/api/process.html#process_process_env) object that is made available to the application
+
+```
+//.env
+MODE=development
+DB_HOST=localhost
+DB_PORT=27017
+DB_USER=root
+DB_PASS=235b599a53bd46c3fb89999b1389ba83
+
+```
+
+
+```
+//server.js
+console.log(process.env.MODE); //development
+console.log(process.env.DB_HOST); //localhost
+console.log(process.env.DB_PORT); //27017
+console.log(process.env.DB_USER); //root
+console.log(process.env.DB_PASS); //235b599a53bd46c3fb89999b1389ba83
+
+```
+
+## How do I use dotenv?
+
+- Create project directory and name it whatever you want.
+- cd into the directory you created and run
+```
+npm init -y
+npm install --save dotenv
+
+``` 
+- Create a .env file and add your settings.
+
+```
+//.env
+
+BASE_URL=https://umuzi.org
+API_KEY=000FBAA4558FF565851A3E104017AB1A
+SERVER_PORT=3000
+USER=umuziadmin
+OBJ={key1:"value1", key2:"value2",key3:"value3",key4:"value4"}
+BOOL=true
+NUM=123456
+STRING= Dumela Lefatshe
+NULL=null
+UNDEFINED=undefined
+ARRAY=[1,2,3,4,5,6]
+
+```
+
+#### &nbsp; Usage
+
+- Create a .js file and start using the module
```

Comment:
Add something like: "You can access your configuration from your code like this"